api/envoy/api/v2/core/protocol.proto

```diff
@@ -85,8 +85,6 @@ message HttpProtocolOptions {
 
   // Total duration to keep alive an HTTP request/response stream. If the time limit is reached the stream will be
   // reset independent of any other timeouts. If not specified, this value is not set.
-  // The current implementation implements this timeout on downstream connections only.
```

Comment:
Please add a 1.15.0/current release note for this.

Reply:
done